packages/@lwc/integration-karma/vitest-scripts/karma-configs/shared/browsers.ts

```diff
@@ -0,0 +1,33 @@
+export const STANDARD_SAUCE_BROWSERS = [
+    {
+        label: 'sl_chrome_latest',
+        browserName: 'chrome',
+        browserVersion: 'latest',
+    },
+    {
+        label: 'sl_firefox_latest',
+        browserName: 'firefox',
+        browserVersion: 'latest',
+        'moz:debuggerAddress': true, // See https://github.com/karma-runner/karma-sauce-launcher/issues/275#issuecomment-1318593354
+    },
+    {
+        label: 'sl_safari_latest',
+        browserName: 'safari',
+        browserVersion: 'latest',
+        platformName: 'macOS 12', // Note: this must be updated when macOS releases new updates
```

Comment:
We've definitely been keeping up with this ... 😅

Reply:
Hopefully that file won't be needed, it's not used right now but I copied it over until that's determined 😄